Source/WebCore/platform/ios/LocalCurrentTraitCollection.h

```diff
@@ -48,6 +48,8 @@ class LocalCurrentTraitCollection {
     RetainPtr<UITraitCollection> m_savedTraitCollection;
 };
 
+WEBCORE_EXPORT UITraitCollection *traitCollectionWithAdjustedIdiom(UITraitCollection *);
```

Comment:
I wonder if we should call this something along the lines of `traitCollectionWithAdjustedIdiomForSystemColors`.

To make it clear this should only be used when determining colors.

Reply:
Makes sense — thanks for the review!